common/dynamicconfig/constants.go

```diff
@@ -452,6 +452,12 @@ to exceed this number will fail with a FailedPrecondition error.`,
 		`MatchingDeletedRuleRetentionTime is the length of time that deleted Version Assignment Rules and
 Deleted Redirect Rules will be kept in the DB (with DeleteTimestamp). After this time, the tombstones are deleted at the next time update of versioning data for the task queue.`,
 	)
+	PollerHistoryTTL = NewNamespaceDurationSetting(
+		"matching.wv.PollerHistoryTTL",
```

Comment:
this doesn't have anything to do with versioning
```suggestion
		"matching.PollerHistoryTTL",
```